### PR TITLE
Ft email confirmation page

### PR DESCRIPTION
**Pull request by Jobsy**
# Description

Describe change or new feature here.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   publish= "build/"
 
 [context.deploy-preview.environment]
```
Comment (nobioma1) [inline netlify.toml]: Can you revert the changes you made to this file, change the local DEV environment URL to the STAGING environment URL? That is the link to the staging env that is used by Netlify for deploy previews.

File: src/config/index.js [hunk 2]
```diff
@@ -1,2 +1,5 @@
 // eslint-disable-next-line import/prefer-default-export
```
Comment (nobioma1) [inline src/config/index.js]: Can you revert the changes to this file? The API_HOST(base_url) should be an environment variable.

File: src/pages/email_confirmation/EmailConfirmation.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  SkewDiv,
+  UnSkewDiv,
+  FlexRowBackground,
+  DesktopImage,
+} from '../../styles/background';
+
+import KnowledgeSVG from '../../assets/images/undraw_knowledge_g5gf.svg';
+
+function EmailConfirmation(props) {
+  console.log('pppppp', props);
```
Comment (nobioma1) [inline src/pages/email_confirmation/EmailConfirmation.js]: Remove console.log

**Review by nobioma1 — CHANGES_REQUESTED** [review]

File: src/pages/email_confirmation/EmailConfirmation.js (hunk 3 above)
Comment (Jobsy) [reply src/pages/email_confirmation/EmailConfirmation.js]: Done!

File: src/config/index.js (hunk 2 above)
Comment (Jobsy) [reply src/config/index.js]: Done!

File: netlify.toml (hunk 1 above)
Comment (Jobsy) [reply netlify.toml]: Done!

**Review by Omulosi — DISMISSED** [review]
Looks okay to me

**Review by ethaeral — APPROVED** [review]
